app/src/main/java/com/github/se/bootcamp/ui/screens/ChallengeScreen.kt

```diff
@@ -1,10 +1,29 @@
 package com.github.se.bootcamp.ui.screens
 
 
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.github.se.bootcamp.ui.navigation.BottomNavigationMenu
+import com.github.se.bootcamp.ui.navigation.LIST_TOP_LEVEL_DESTINATION
+import com.github.se.bootcamp.ui.navigation.NavigationActions
 
 @Composable
-fun ChallengeScreen() {
-    Text(text = "Challenge with Friends Screen")
+fun ChallengeScreen(navigationActions: NavigationActions) {
+    Scaffold (
+        bottomBar = {
+            BottomNavigationMenu(
+                onTabSelect = { route -> navigationActions.navigateTo(route) },
+                tabList = LIST_TOP_LEVEL_DESTINATION,
+                selectedItem = navigationActions.currentRoute())
+        },
+        content = { pd ->
+            Text(
+                modifier = Modifier.padding(pd),
+                text = "Challenge with Friends Screen"
```

Comment:
It's a good habit to develop to avoid writing hard coded string in the code, and favor the use of String.xml resources. This will makes it easier to later [localize](https://developer.android.com/guide/topics/resources/localization) the application. It also makes the app more maintainable as it is easy to modify one text, without having to remember to modify every places that are displaying the hard coded text.